src/NewColumnView.elm

```diff
@@ -84,14 +97,14 @@ view model =
                         ]
                         [ text "Add New Column" ]
                     ]
-                , div [ class "prog-form--warning" ] [ text (newColumnWarning model) ]
+                , validationWarningDiv
                 , button
                     [ class "button button--secondary"
```

Comment:
I know it's outside the diff and I'm sorry.
Please can we have these take the --wider modifier as well as the save button